### PR TITLE
CRM-19630 optionally allow bi-directional relationships

### PR DESCRIPTION
**Pull request by konadave**
* [CRM-19630: Optionally allow bi-directional relationships](https://issues.civicrm.org/jira/browse/CRM-19630)

### CODE REVIEW

**Comment by yashodha** [comment]
@konadave : xml changes for the added field seem to be missing in the PR.

**Comment by jitendrapurohit** [comment]
Cherry-picked the commits and created a new PR at https://github.com/civicrm/civicrm-core/pull/10156 removing the conflicts and also added test.

File: templates/CRM/Admin/Form/RelationshipType.tpl [hunk 1]
```diff
@@ -61,6 +61,12 @@
                 <td class="label">{$form.is_active.label}</td>
                 <td>{$form.is_active.html}</td>
             </tr>
+            <tr class="crm-relationship-type-form-block-is_bidirectional">
+                <td class="label">{$form.is_bidirectional.label}</td>
+                <td>{$form.is_bidirectional.html}<br />
+                  <span class="description">{ts}Can a 'B to A' relationship be created when their is's an existing 'A to B' relationship?{/ts}</span>
```
Comment (seanmadsen) [inline templates/CRM/Admin/Form/RelationshipType.tpl]: grammar/spelling change suggestion:  change `their is's` to `there is`

**Comment by eileenmcnaughton** [comment]
Closing in favour of #10156

**Pull request closed without merging**